.jshintrc

```diff
@@ -1,13 +1,14 @@
 {
-	"boss": true,
-	"curly": true,
-	"eqeqeq": true,
-	"immed": true,
-	"latedef": true,
-	"newcap": true,
-	"noarg": true,
-	"node": true,
-	"strict": true,
-	"sub": true,
-	"undef": true
-}
\ No newline at end of file
+  "boss": true,
```

Comment:
I'm not familiar with .editorconfig, but I can try adding it later in this branch. 2 spaces everywhere right?